mappings/1.3-pre-07261249#1.3.1.tinydiff

```diff
@@ -2131,16 +2131,16 @@ c	net/minecraft/unmapped/C_6908971	net/minecraft/world/biome/layer/DownfallBlend
 		p	2		p_2	
 		p	3		p_3	
 		p	4		p_4	
-c	net/minecraft/unmapped/C_6922944	net/minecraft/unmapped/C_6922944	
+c	net/minecraft/unmapped/C_6922944	net/minecraft/item/CookieItem	
 	m	(IIZI)V	<init>	<init>	
-		p	1		p_1	
-		p	2		p_2	
-		p	3		p_3	
-		p	4		p_4	
-c	net/minecraft/unmapped/C_7017824	net/minecraft/unmapped/C_7017824	
-	f	I	f_3556592	f_3556592	
-	f	I	f_3762256	f_3762256	
-	f	I	f_0382768	f_0382768	
+		p	1		id	
+		p	2		hungerPoints	
+		p	3		canBeCooked	
+		p	4		maxStackSize	
+c	net/minecraft/unmapped/C_7017824	net/minecraft/world/Coord	
```

Comment:
Not too big a fan of this but I can't find any usages of this class at all so who knows what it's supposed to be. `Vec3i` and `BlockPos` are both already taken.